app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/gestures/CesiumOnMapGestureListener.java

```diff
@@ -0,0 +1,67 @@
+package com.teamagam.gimelgimel.app.view.viewer.gestures;
+
+import android.app.Fragment;
+import android.webkit.ValueCallback;
+
+import com.teamagam.gimelgimel.app.view.fragments.dialogs.SendGeographicMessageDialog;
+import com.teamagam.gimelgimel.app.view.viewer.GGMap;
+import com.teamagam.gimelgimel.app.view.viewer.data.geometries.PointGeometry;
+
+/**
+ * Created by CV on 6/20/2016.
+ */
+public class CesiumOnMapGestureListener implements OnMapGestureListener {
```

Comment:
Move class to cesium package !resolve 
extend from SimpleOnMapGestureListener to avoid empty implementations